nist/curve.go

```diff
@@ -252,6 +252,18 @@ func (c *curve) Point() abstract.Point {
 	return p
 }
 
+func (p *curvePoint) Set(P abstract.Point) abstract.Point {
+	p.x = P.(*curvePoint).x
+	p.y = P.(*curvePoint).y
+	return p
+}
+
+func (p *curvePoint) Clone() abstract.Point {
+	// TODO test if we really created a new Point
+	// changing original point doesn't change cloned
```

Comment:
Apparently the tests are already verifying that invariant so the comment can be removed i guess.